charts/airbyte/templates/env-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: airbyte-env
+data:
+  AIRBYTE_VERSION: {{ .Chart.AppVersion }}
+  API_URL: /api/v1/
+  AWS_ACCESS_KEY_ID: {{ .Values.minio.accessKey.password }}
+  AWS_SECRET_ACCESS_KEY: {{ .Values.minio.secretKey.password }}
+  CONFIG_ROOT: /configs
+  DATA_DOCKER_MOUNT: airbyte_data
+  DATABASE_DB: {{ .Values.postgresql.postgresqlDatabase }}
+  DATABASE_HOST: {{ include "airbyte.database.host" . }}
+  DATABASE_PASSWORD: {{ .Values.postgresql.postgresqlPassword }}
+  DATABASE_PORT: "5432"
+  DATABASE_URL: jdbc:postgresql://airbyte-postgresql:5432/{{ .Values.postgresql.postgresqlDatabase }}
+  DATABASE_USER: {{ .Values.postgresql.postgresqlUsername }}
+  DB_DOCKER_MOUNT: airbyte_db
+  FULLSTORY: enabled
+  GCP_STORAGE_BUCKET: ""
+  GOOGLE_APPLICATION_CREDENTIALS: ""
+  INTERNAL_API_HOST: {{ include "common.names.fullname" . }}-server:{{ .Values.server.service.port }}
+  IS_DEMO: "false"
+  LOCAL_ROOT: /tmp/airbyte_local
+  LOG_LEVEL: INFO
+  OPENREPLAY: enabled
+  PAPERCUPS_STORYTIME: enabled
+  RESOURCE_CPU_LIMIT: ""
+  RESOURCE_CPU_REQUEST: ""
+  RESOURCE_MEMORY_LIMIT: ""
+  RESOURCE_MEMORY_REQUEST: ""
+  RUN_DATABASE_MIGRATION_ON_STARTUP: "true"
+  S3_LOG_BUCKET: airbyte-dev-logs
+  S3_LOG_BUCKET_REGION: ""
+  S3_MINIO_ENDPOINT: http://airbyte-minio:9000
```

Comment:
Should this be `http://airbyte-minio-svc:9000`? 
Ran into issues where it couldn't execute to this endpoint